test_scripts/API/Restructuring_OnResetTimeout/001_BC_OnResetTimeout_Success_flow.lua

```diff
@@ -0,0 +1,50 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0189-Restructuring-OnResetTimeout.md
+--
+-- Description:
+-- In case:
+-- 1) RPC is requested
+-- 2) Some time after receiving RPC request on HMI is passed
+-- 3) HMI sends BC.OnResetTimeout(resetPeriod = 15000) to SDL
+-- 4) HMI sends response in 11 seconds after receiving request
```

Comment:
I'm assuming `defaultTimeout = 10 seconds` specified in smartDeviceLink.ini, and this is the reason 11 is chosen in test scripts. Without BC.OnResetTimeout(resetPeriod = 15000), the response from HMI at 11 will be out of the timeout windows, thus will be ingored. But with BC.OnResetTimeout(resetPeriod = 15000), 11 is inside the timeout window, thus is processed.